### PR TITLE
add automation tests in SolidTests.cs

### PR DESCRIPTION
**Pull request by zora-wang**
Purpose:

add/modify automation tests that were written previously in SolidTests.cs

Reviewer:
@riteshchandawar  @Benglin

### CODE REVIEW

File: test/Libraries/RevitIntegrationTests/SolidTests.cs [hunk 1]
```diff
@@ -4,96 +4,118 @@
...
+            RunCurrentModel();
+
+            AssertNoDummyNodes();
 
-            //var fec = new FilteredElementCollector( DocumentManager.Instance.CurrentDBDocument );
-            //fec.OfClass(typeof(GenericForm));
+            Assert.AreEqual(30, ViewModel.Model.CurrentWorkspace.Nodes.Count);
+            Assert.AreEqual(53, ViewModel.Model.CurrentWorkspace.Connectors.Count());
 
-            ////verify one loft created
```
Comment (riteshchandawar) [inline test/Libraries/RevitIntegrationTests/SolidTests.cs]: Where is the verification code?

**Comment by riteshchandawar** [comment]
There is one test case missing verification, rest LGTM.

File: test/Libraries/RevitIntegrationTests/SolidTests.cs (hunk 1 above)
Comment (zora-wang) [inline test/Libraries/RevitIntegrationTests/SolidTests.cs]: @riteshchandawar , Loft.dyn contains warning, and it cannot be tested now. so this method has been commentted

**Comment by riteshchandawar** [comment]
LGTM!

@Benglin Please help us to merge this in.

Thanks,
Ritesh

**Pull request merged**